src/test/MappingsClient.test.ts

```diff
@@ -214,7 +214,7 @@ describe("mappings Client", () => {
 
     mapping = await mappingsClientNewBase.updateMapping("auth", "iModelId", "mappingId", newMapping);
     expect(fetchStub.calledWith(
-      "https://api.bentley.com/insights/reporting/datasources/imodels/iModelId/mappings/mappingId",
+      "BASE/datasources/imodels/iModelId/mappings/mappingId",
```

Comment:
What happened here?

Reply:
You are reusing the same `fetchStub` multiple times in this test, so you do not know which method called `fetchStub` with what parameters.

Reply:
I am reusing the fetch stub. I first fetch with the client that has the default base path ant then with the client that has ```BASE``` base path. When writing tests I noticed that this expect was not using the ```BASE``` path so I changed it (was epecting the same thing twice). Overall these tests should be remade using ```getcall(0).args``` because right now if they fail the message is ```Expected false be eqial to true```. Not sure if this should be done in this PR though.

Reply:
What I wanted to point out that this change does not make sense. The test was not failing with the wrong expectation.

Reply:
Now changing base path is being handled in another test